modules/graphdb/templates/04_gdb_conf_overrides.sh.tpl

```diff
@@ -37,12 +37,20 @@ log_with_timestamp "Writing configuration files"
 # graphdb.external-url.enforce.transactions: determines whether it is necessary to rewrite the Location header when no proxy is configured.
 # This is required because when working with the GDB transaction endpoint it returns an erroneous URL with HTTP protocol instead of HTTPS
 if [ "${node_count}" -eq 1 ]; then
+
+  if [ "${disable_agw}" = "true" ]; then
+  EXTERNAL_URL="https://${graphdb_external_address_fqdn}${context_path}/"
+  else
+  EXTERNAL_URL="https://${graphdb_external_address_fqdn}"
```

Comment:
Why can't we have context path here as well?

Reply:
trying to avoid errors for users. If they put context path without the 'disable_agw', the user_data script breaks, since the graphdb.properties is not correct and it throws unrelated erros. 
With this validation I am trying to remove user-error. If there is no "disable_agw" there will be no context path.
Less room for errors IMO.

Reply:
Are we really going to limit functionality just because something in the user data scripts breaks? Wouldn't it be better to fix the scripts instead, so that `context_path` is available regardless of whether `disable_agw` is enabled or not? The way the variables are currently set up, they seem independent, and in reality, they should be. There's no inherent connection between `context_path` and whether an external application gateway exists. It's just as common to have GraphDB behind a context path as it is to have it at the root, and this shouldn't depend on the presence or lack of an external gateway.